### DISCUSSION TITLE
蚂蚁金服的SOFABolt是不是和rsocket很像

### DISCUSSION THREAD

**Discussion by jimium** [alibaba/alibaba-rsocket-broker#101] [General]
sofabolt的api看起来和rsocket理念差不多是吗

**Comment by linux-china** [comment]
有使用过 Sofa bolt的同学来回答一下，就我了解的，还不一样 :)     Sofa还是面向服务架构，RSocket Broker主要是协议 + 事件驱动 + Reactive异步服务接口。 



>> SOFA 是蚂蚁金服自主研发的金融级分布式中间件，包含了构建金融级云原生架构所需的各个组件，包括微服务研发框架，RPC 框架，服务注册中心，分布式定时任务，限流/熔断框架，动态配置推送，分布式链路追踪，Metrics监控度量，分布式高可用消息队列，分布式事务框架，分布式数据库代理层等组件，是一套分布式架构的完整的解决方案，也是在金融场景里锤炼出来的最佳实践。

**Comment by AtticusLv** [comment]
rsocket异步非阻塞
